src/modules/csm/tree.py

```diff
@@ -1,18 +1,35 @@
 import json
 from dataclasses import dataclass
-from typing import Self, Sequence
+from typing import Literal, Self, Sequence
 
 from hexbytes import HexBytes
-from oz_merkle_tree import StandardMerkleTree
+from oz_merkle_tree import Dump, StandardMerkleTree
 
 from src.modules.csm.types import RewardTreeLeaf
+from src.providers.ipfs.cid import CID
+
+type StateCID = CID | Literal[""]
+
+
+class TreeDump(Dump[RewardTreeLeaf]):
+    stateCID: StateCID
```

Comment:
to snake_case

Reply:
The Dump type is in camelSnake as it forced by OZ original library. I don't see a reason to mess with different styles here. So all dumps intended to be JSON-serialized are currently in camelCase. I don't want to add a conversion pipeline.